charts/ingress-nginx/templates/controller-poddisruptionbudget.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (and (not .Values.controller.autoscaling.enabled) (gt (.Values.controller.replicaCount | int) 1)) }}
+{{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (and (not .Values.controller.autoscaling.enabled) (gt (.Values.controller.replicaCount | int) 1))  (and .Values.controller.keda.enabled (gt (.Values.controller.keda.minReplicas | int) 1)) }}
```

Comment:
This only check if keda is enabled and then deploy the PDB, should we add a keda config as well?